src/main/java/uk/gov/digital/ho/hocs/search/api/CaseDataService.java

```diff
@@ -40,6 +40,10 @@ public class CaseDataService {
 
     private final int resultsLimit;
 
+    private final Set<String> correspondentFields = Set.of("allCorrespondents", "currentCorrespondents");
```

Comment:
I don't like this but the only alternative is reflection or creating specific methods inside the ES client.